src/app/Dashboard/components/DashboardContainer.js

```diff
@@ -94,6 +109,69 @@ export class DashboardContainer extends Component {
     });
   };
 
+  handleClick = index => (event) => {
+    if (event.target.value === 'Select All') {
```

Comment:
I think this should be `select all` instead of `Select All` looking at the value that comes from the target object